docs/examples/balltree-kdtree-performance.ipynb

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
<p>1M neighbors is a big number, it's important to know what the original dimensions are.</p><p> </p><p>How about finding just a handful of neighbors &lt; 20, that is more realistic use case, I think. </p><p>We can keep both cases.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/587/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I think the 1M neighbors is meant to showcase the difference in query performance, since for < 20 they might seem the same.

Reply:
Yeah, for 5 cases it’s like around 0 seconds for both of them. Pretty much the same as the 1000.